app/scripts/utils/utils.js

```diff
@@ -87,7 +87,7 @@ export function isCoreVersionCompatible (minVersion, actualVersion) {
   const v1 = minVersion.split('.')
   const v2 = actualVersion.split('.')
 
-  if (v1[0] === v2[0]) {
+  if (v1[0] <= v2[0]) {
```

Comment:
Would this cater for resolving of the Bug logged within Jira? or is this till a bigger issue that needs resolving?
https://jembiprojects.jira.com/browse/OHM-741